File: Ruby/fizzbuzz.rb
def fizzBuzz(upto)
for num in 1..upto
  if num % 3 === 0 and num % 5 === 0  
   puts "FizzBuzz"
elsif num % 3 === 0
   puts "Fizz"
elsif num % 5 === 0
   puts "Buzz"
else
    puts num
end
end
end

fizzBuzz 100

File: Python/fizzbuzz.py
def fizzbuzz(upto):
    for num in range(1,upto + 1):
        if num%3 == 0 and num%5 == 0:
            print("FizzBuzz")
        elif num%3 ==0:
            print('Fizz')
        elif num%5==0:
            print("Buzz")
        else:
            print(num)

fizzbuzz(100)